src/AccessControl/cAccessControl.c

```diff
@@ -425,7 +425,7 @@ static PyExtensionClass ZopeSecurityPolicyType = {
 	0,					/* tp_itemsize	*/
 	/* Standard methods 	*/
 	(destructor) ZopeSecurityPolicy_dealloc,/* tp_dealloc	*/
-	NULL,					/* tp_print	*/
+	0,  					/* tp_print	*/
```

Comment:
```suggestion
	0,  					/* tp_vectorcall_offset	*/
```
We should change the comment to reflect the new use of this field.